### DISCUSSION TITLE
The same command runs, why do some of them have errors

### DISCUSSION THREAD

**Discussion by KmBooy** [Chia-Network/chia-blockchain#6139] [Support]
![image](https://user-images.githubusercontent.com/35790216/119913445-84d9e380-bf90-11eb-852b-1ebfa7897e98.png)
![image](https://user-images.githubusercontent.com/35790216/119913534-b652af00-bf90-11eb-9fb6-93c3c2df96e2.png)

The same command runs, why do some of them have errors

 Version：windows 10 Professional
Chia Version：1.1.6

**Comment by BrandtH22** [comment]
@KmBooy , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note this is a duplicate discussion.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
